EIPS/eip-634.md

```diff
@@ -1,8 +1,9 @@
 ---
 eip: 634
 title: Storage of text records in ENS 
-author: Richard Moore <me@ricmoo.com>
+author: Richard Moore (@ricmoo)
 type: Standards Track
+discussions-to: https://github.com/ricmoo/EIPs/issues/2
```

Comment:
Don't wanna nitpick, but there would be a slight preference for https://github.com/ethereum/EIPs/issues or ethereum-magicians.org (as most EIPs use these two). It is up to you however.

Reply:
Oh, I don't mind doing that at all, but then there would be 2 "issue" numbers for the single EIP, since PR and issues share the same ID space. Is that ok?

Reply:
Yes, a lot of the EIPs do this.

Reply:
Done. :)